studiopress/genesis-custom-blocks
# Bump the version to 1.2.0

#### Changes
* Bumps the version to 1.2.0
* Adds a CHANGELOG entry
* Adds Rob's new screenshot of the Template Editor to the wp.org screenshots

#### Testing instructions
1. `cd` to this plugin
1. `git fetch`
1. `git checkout bump/version-1.2.0`
1. `php bin/verify-versions.php`
1. Expected: `Success! All of the version numbers are the same, and there's a CHANGELOG.md entry.`
